### PR TITLE
Updates section on home page

### PR DESCRIPTION
**Pull request by sambulance**
Improvements to the home page to show new pages and recent edits.

Addresses #6

### CODE REVIEW

File: inc/updates.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	}
+	$output .= '</ol>';
+
+	echo $output;
+}
+
+function edited_pages_only( $where = '' ) {
+	global $wpdb;
+
+	$where .= $wpdb->prepare( " AND $wpdb->posts.post_date != $wpdb->posts.post_modified AND $wpdb->posts.post_type = %s", 'page' );
```
Comment (sambulance) [inline inc/updates.php]: Hey @mattheu, please can I get your opinion here?

This query selects posts that have been edited. This is done by comparing `post_date` and `post_modified` values. 

The problem is `$wpdb->prepare()` requires a placeholder argument, so I've added the post type query to add in a placeholder element. This feels messy and is duplicating part of the query.

Do we need to use `$wpdb->prepare()` when no placeholders are required? Is it okay/cleaner to just pass this as a string? That also feels wrong as it isn't being sanitised.

Thoughts?

Comment (mattheu) [inline inc/updates.php]: I think it's only necessary to use prepare when you're passing I trusted data in. If it's a string you are hard coding, then you can trust that 100% and you don't need to sanitize!

Comment (sambulance) [inline inc/updates.php]: Thanks @mattheu ... I thought that but didn't feel like the right solution. Will go with it. Thanks :)

Comment (mattheu) [inline inc/updates.php]: 👍 Just a note - my previous comment I wrote 'trusted' when I meant 'untrusted'... pretty bit mix up :) but I think you knew what I mean!

**Comment by mattheu** [comment]
I'm wondering if you could query for post revisions instead and avoid having to do the custom SQL?

**Comment by sambulance** [comment]
@mattheu Had a look into revisions, but they aren't publicly queriable so we can't use them here. Not keen on the SQL either so open to suggestions.

**Comment by mattheu** [comment]
I'm going to go ahead and merge this for now. Lets circle back though, as I think we can do something with revisions for this. Would a custom WP_Query will work even though their not publicly queryable?

**Comment by mattheu** [comment]
New issue here: https://github.com/humanmade/hm-handbook-theme/issues/78

**Comment by mattheu** [comment]
I wonder whether we should remove the page title from the home page?  Feels a bit odd.

Perhaps we have a front page template file and move the loading of the updates template part into that?

**Pull request merged**